src/components/ModalConfirmCode.tsx

```diff
@@ -0,0 +1,104 @@
+import { connected } from 'process';
+import React,  {FC, useEffect, useState} from 'react';
+import './ModalConfirmCode.scss';
+
+interface CardProps {
+  active: boolean;
+  setActive: (bool: boolean) => void;
+}
+
+const ModalConfirmCode: FC<CardProps> = 
+  ({
+    active, 
+    setActive,
+  }) => {
+  
+  const [code, setCode] = useState('');
+  const [codeDirty, setCodeDirty] = useState(false);
+  const [codeError, setCodeError] = useState('The input field must be filled');
+  const [formValid, setFormValid] = useState(false);
+
+  useEffect(() => {
+    setFormValid(!codeError);
+  }, [codeError])
+
+  const sendCode = () => {
+    setActive(false);
+  }
+
+  const blurHandler = () => {
+    setCodeDirty(true);
+  }
+
+  const codeHandler = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setCode(e.target.value)
```

Comment:
you use e.target.value multiple times. Please create constant in the beginning of the codeHandler:

const {value} = e.target

Reply:
done